### PR TITLE
emscripten: force resize event when pixel ratio changes

### PR DESCRIPTION
**Pull request by dos1**
Without this, applications can't react to changed canvas size on window zoom.

https://bugzilla.libsdl.org/show_bug.cgi?id=4512

### CODE REVIEW

File: src/video/emscripten/SDL_emscriptenevents.c [hunk 1]
```diff
@@ -611,6 +611,10 @@ Emscripten_HandleResize(int eventType, const EmscriptenUiEvent *uiEvent, void *u
                 emscripten_set_element_css_size(NULL, w, h);
             }
 
+            /* force the event to trigger, so pixel ratio changes can be handled */
```
Comment (Daft-Freak) [inline src/video/emscripten/SDL_emscriptenevents.c]: We should probably only do this if the ratio has changed.

Comment (dos1) [reply src/video/emscripten/SDL_emscriptenevents.c]: Won't that be already handled by the browser? (as in, won't this code be executed only if there's actual change of the window size or pixel ratio?)

Comment (Daft-Freak) [reply src/video/emscripten/SDL_emscriptenevents.c]: It's possible for the canvas size to not change when the window resizes (min/max width, or even the default where the canvas doesn't change size).

Comment (dos1) [reply src/video/emscripten/SDL_emscriptenevents.c]: Ah, right, the handler's attached to a window... Pity that `ResizeObserver` is still Chrome-only :( OK then, I'll update the patch.

**Comment by Daft-Freak** [comment]
Thanks. (Delayed as I forgot that github doesn't send notifications for PR updates...)

**Pull request merged**